pixeltable/ext/functions/yolox.py

```diff
@@ -22,14 +22,28 @@
 @pxt.udf(batch_size=4)
 def yolox(images: Batch[PIL.Image.Image], *, model_id: str, threshold: float = 0.5) -> Batch[dict]:
     """
-    Runs the specified YOLOX object detection model on an image.
+    Computes YOLOX object detections for the specified image. `model_id` should reference one of the models
+    defined in the [YOLOX documentation](https://github.com/Megvii-BaseDetection/YOLOX).
 
     YOLOX support is part of the `pixeltable.ext` package: long-term support is not guaranteed, and it is not
```

Comment:
Maybe we should have an 'ext' section in the docs? I expect that to get more voluminous.